docs/pages/slimbanner.js

```diff
@@ -0,0 +1,472 @@
+// @flow strict
+import { type Node } from 'react';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import docgen, { type DocGen } from '../components/docgen.js';
+
+export default function SlimBannerPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader
+        name={generatedDocGen?.displayName}
+        description={generatedDocGen?.description}
+        defaultCode={`
+<SlimBanner
+  type="info"
+  message="Idea Pins are now available across platforms."
+  iconAccessibilityLabel="Information"
+/>
+`}
+      />
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+- When displaying section-level information to the user.
+- When providing persistent messaging/guidance for specific elements or areas within a surface or page.
+- When providing messaging/guidance on information-dense screens where there is limited space.
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+- When displaying information that pertains to the whole page and is of the highest priority. Use [Callout](/callout) instead.
+- When interacting with the SlimBanner is required for the user to proceed with a task or flow. Use [Modal](/modal) instead.
+- When describing the function of an interactive element that doesn’t have a text label. Use [Toast](/toast) instead.
```

Comment:
Toast? Not sure I understand this recommendation

Reply:
Should it be Tooltip?

Reply:
it should be tooltip. Sorry, my mistake.

Reply:
DONE